scout/commands/update/genes.py

```diff
@@ -133,12 +133,12 @@ def genes(build, downloads_folder, api_key):
 
     # If required resources are missing, download them to a temporary file
     if downloads_folder is None:
-        with tempfile.TemporaryDirectory() as tempdir:
+        with tempfile.TemporaryDirectory(dir=os.getcwd()) as tempdir:
             try:
                 download_resources(tempdir, api_key, builds)
             except Exception as ex:
                 LOG.error(ex)
-        fetch_downloaded_resources(resources, tempdir, builds)
+            fetch_downloaded_resources(resources, tempdir, builds)
```

Comment:
This is the fix

Reply:
Thanks, I never quite got why on earth it should survive outside the with block anyway. Python variable life is a little involved, but this is clearer regardless..